xDnsServer.psd1

```diff
@@ -1,6 +1,6 @@
 @{
 # Version number of this module.
-ModuleVersion = '1.7.0.0'
+ModuleVersion = '1.7.0.1'
```

Comment:
Can you please remove these changes to the manifest?
The manifest will be updated when the module is released.